blog.html

```diff
@@ -0,0 +1,251 @@
+                <img src="images/connect.svg" width="24" height="24" alt="connect">
+                Connect
+            </a>
+        </div>
+    </header>
+    <section class="blog-intro">
+        <h2 class="blog-intro__title">Our blog</h2>
+        <ul class="blog-intro__nav">
+            <li href="#" class="blog-intro__item">Home</li>
+            <li href="#" class="blog-intro__item">Our blog</li>
+            <li href="#" class="blog-intro__item">The Basilisks NFT</li>
+        </ul>
+    </section>
+     <main class="blog-wrapper">
+        <aside class="blog-sidebar">
+            <div class="blog-sidebar__block">
+                <h3 class="blog-sidebar__title">Categories</h3>
+                <ul class="categories__wrapper">
+                    <li class="categories__item"><a href="#">Our blog <span>(12)</span></a></li>
+                    <li class="categories__item"><a href="#">Metaverse <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Gaming <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Art <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Robot <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Launchpad <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Crypto <span>(430)</span></a></li>
+                </ul>
+            </div>
+            <div class="blog-sidebar__block">
+                <h3 class="blog-sidebar__title">Tag</h3>
+                <ul class="tags__wrapper">
+                    <li class="tags__item"><a href="#">NFT</a></li>
+                    <li class="tags__item"><a href="#">ICO</a></li>
+                    <li class="tags__item"><a href="#">Robot</a></li>
+                    <li class="tags__item"><a href="#">Crypto</a></li>
+                    <li class="tags__item"><a href="#">Launchpad</a></li>
+                    <li class="tags__item"><a href="#">Metaverse</a></li>
+                </ul>
+            </div>
+        </aside>
+        <div class="blog-content">
+            <figure class="blog-content__photo">
+                <img src="images/blog-photo.jpg" width="920" height="557" alt="dragon with man">
+                <figcaption class="blog-content__caption">Photo caption</figcaption>
+            </figure>
+            <article class="blog-content__info">
+                <h2 class="subtitle">The Basilisks NFT</h2>
+                    <a href="#" class="btn btn__img btn__color">Gaming</a>
+                    <div class="blog__info">
+                        <p class="blog__name">Tony Nguyen</p>
+                        <time datetime="2022-06-20" class="blog__date">20 Jun 2022</time>
+                    </div>
+                    <p class="blog__text">The Basilisks that <mark>players</mark> collect are represented with NFTs on Ethereum’s blockchain with real-world value. Players use the 
```

Comment:
No need classes on the `<p> <a> <ul>` (except ul.blog__list--mark with blue bullets) - it's  would be 
WYSIWYG content